samples/siwx91x_ota/prj.conf

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2025 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Network Stack Configuration
+CONFIG_NETWORKING=y
+CONFIG_NET_MGMT=y
+CONFIG_NET_TCP=y
+CONFIG_NET_CONNECTION_MANAGER=y
+
+# IPv4/IPv6 Support
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_ARP=y
+CONFIG_NET_DHCPV4=y
+CONFIG_NET_DHCPV6=y
+
+# WiFi Configuration
+CONFIG_WIFI=y
+
+# Memory and Threading
+CONFIG_MAIN_STACK_SIZE=4098
+CONFIG_INIT_STACKS=y
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+
+# CMSIS Configuration
+CONFIG_CMSIS_V2_MUTEX_MAX_COUNT=10
+CONFIG_CMSIS_V2_EVT_FLAGS_MAX_COUNT=10
+
+# Network Parameters
+CONFIG_NET_TCP_MAX_RECV_WINDOW_SIZE=10240
+
+# Socket Support
+CONFIG_REQUIRES_FULL_LIBC=y
+CONFIG_NET_SOCKETS=y
+
+# HTTP Client Configuration
+CONFIG_HTTP_CLIENT=y
+CONFIG_HTTP_PARSER_URL=y
+CONFIG_NET_MGMT_EVENT=y
+
+# TLS Security Configuration
+CONFIG_MBEDTLS=y
+CONFIG_MBEDTLS_BUILTIN=y
+CONFIG_MBEDTLS_ENABLE_HEAP=y
+CONFIG_MBEDTLS_HEAP_SIZE=60000
+CONFIG_TLS_CREDENTIALS=y
+CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
+CONFIG_MBEDTLS_SSL_MAX_CONTENT_LEN=8000
+
+# TLS Protocol and Cipher Configuration
+CONFIG_MBEDTLS_TLS_VERSION_1_2=y
+CONFIG_MBEDTLS_SHA256=y
+CONFIG_MBEDTLS_CIPHER_AES_ENABLED=y
+CONFIG_MBEDTLS_CIPHER_ALL_ENABLED=y
+CONFIG_MBEDTLS_CIPHER_CHACHA20_ENABLED=y
+CONFIG_MBEDTLS_POLY1305=y
```

Comment:
`CONFIG_HASH_ALL_ENABLED`? So you won't have to select individually SHA256 and POLY1305?

I am not a specialist of TLS. I believe CHACHA20+POLY1305 is indeed very common. However, maybe we don't want to fine tune the protocols we compile in this sample since this is not the main focus of the sample.